pyscf/adc/uadc.py

```diff
@@ -4068,7 +4632,12 @@ def __init__(self, adc):
     matvec = ip_adc_matvec
     compute_trans_moments = ip_compute_trans_moments
     get_trans_moments = get_trans_moments
-    get_spec_factors = get_spec_factors
+    get_properties = get_properties
+
+    spec_analyze = spec_analyze
+    analyze_eigenvector = analyze_eigenvector_ip
+    analyze = analyze
+    compute_dyson_orb = compute_dyson_orb
```

Comment:
Please make sure all RADC changes are applied to UADC.

Reply:
Done